index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="../css/styles.css" type="text/css">
+</head>
+<body>
+    <header>
+        <img src="../src/assets/images/header-shapes-mobile@2x.svg" alt="background" class="bg_shape">
+        <nav>
+            <ul class="flex-nav">
+                <li class="logo">
+                    <a href="#"><img alt="logo" src="../src/assets/images/my_logo.png"></a>
+                </li>
+                <li class="menu">
+                    <a href=""><img alt="Menu" src="../src/assets/images/img_icon.png"></a>
+                </li>
+            </ul>
+        </nav>
+    </header>
```

Comment:
- [OPTIONAL] Kindly make sure that the background image on the `header` section looks exactly like the provided Figma design.

  - ### Your design
  ![his log](https://user-images.githubusercontent.com/87197412/185671930-5947bcb8-6868-44f9-a5bf-78342b4753b7.PNG)


   - ### Figma design
  ![real logo](https://user-images.githubusercontent.com/87197412/185671967-6f11a922-b95a-416e-91d8-f44f3be1ade8.PNG)